app/src/main/res/layout/pdp_fragment.xml

```diff
@@ -17,18 +17,18 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        android:src="@drawable/image_product" />
+        tools:src="@drawable/image_product" />
 
     <View
         android:id="@+id/view"
         android:layout_width="0dp"
         android:layout_height="0dp"
-        app:layout_constraintTop_toBottomOf="@id/iv_product"
+        android:layout_marginTop="@dimen/normal_150"
+        android:background="@drawable/ic_rectangle_1"
```

Comment:
can we rename `rectangle_1` to something more concrete?
E.g. `pdp_rounded_background` or other option that you like

Reply:
done

Reply:
done